SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -196,7 +196,7 @@ public static CareCaseData ToCareCaseData(this CaseSubmission caseSubmission, Li
                 CaseFormTimestamp = caseSubmission.SubmittedAt?.ToString("yyyy-MM-dd") ?? DateTime.Now.ToString("yyyy-MM-dd"),
                 FormName = caseSubmission.FormId,
                 DateOfBirth = resident.DateOfBirth?.ToString("dd/MM/yyyy"),
-                DateOfEvent = caseSubmission.CreatedAt.ToString("yyyy-MM-dd"),
+                DateOfEvent = caseSubmission.DateOfEvent?.ToString("yyyy-MM-dd") ?? caseSubmission.CreatedAt.ToString("yyyy-MM-dd"),
```

Comment:
how will this play with the need to order by time as well as date?

Reply:
In `CaseRecordsUseCase.cs` there is a function `SortData` that handles the sorting in the backend

Reply:
Let me check, if ISO is possible anywhere I always prefer to use it

Reply:
Using ISO now which will play better with date and time sorting! (plus should always be parseable)